BoxContentSDK/BoxContentSDK/Requests/BOXUserAvatarRequest.m

```diff
@@ -0,0 +1,104 @@
+//
+//  BOXUserAvatarRequest.m
+//  BoxContentSDK
+//
+
+#import "BOXRequest_Private.h"
+#import "BOXUserAvatarRequest.h"
+
+#import "BOXAPIDataOperation.h"
+
+@interface BOXUserAvatarRequest ()
+
+@property (nonatomic, readonly, strong) NSOutputStream *outputStream;
+
+@end
+
+@implementation BOXUserAvatarRequest
+
+- (instancetype)initWithUserID:(NSString *)userID
+{
+    if (self = [super init]) {
+        _userID = userID;
+        _outputStream = [[NSOutputStream alloc] initToMemory];
+
+        _avatarType = BOXAvatarTypeUnspecified;
+    }
+
+    return self;
+}
+
+- (BOXAPIOperation *)createOperation
+{
+    NSURL *URL = [self URLWithResource:BOXAPIResourceUsers
+                                    ID:self.userID
+                           subresource:BOXAPISubresourceAvatar
+                                 subID:nil];
+
+    NSMutableDictionary *queryParameters = [NSMutableDictionary dictionary];
+
+    NSString *avatarTypeString = nil;
```

Comment:
minor - we could move this to be a helper to make the `createOperation` method cleaner